### PR TITLE
iOS sdk release 6.9.0

### PR DESCRIPTION
**Pull request by Ghasfj**
I have added the release notes for the incoming release. 

Additionally changes also added upon request for our pages.

### CODE REVIEW

**Comment by j9t** [comment]
(Please loop in @itay1313 or me when ready for review.)

**Comment by Ghasfj** [comment]
@j9t @itay1313 This should be released in about 2 hours. I sent a direct message.

File: pages/Documents/MessagingChannels/MobileAppMessagingSDKforiOS/ReleaseNotes/Regular/6.9.0.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+- Added support for Bluetooth keyboards and key commands for accessibility users.
+- The loading screen is now announced for screen reader users.
+- Support for future messaging backends increased.
+
+#### New configurations
+
+##### inputTextViewPlaceholderColor
+- **Description:** Edit the color of the input text placeholder.
+- **Type:** UIColor
+- **Default Value:**  nil
```
Comment (j9t) [inline pages/Documents/MessagingChannels/MobileAppMessagingSDKforiOS/ReleaseNotes/Regular/6.9.0.md:42]: Use 1 instead of 2 spaces before “nil”

**Review by j9t — CHANGES_REQUESTED** [review]

**Comment by j9t** [comment]
@Ghasfj, did I miss something or did you coordinate with us beforehand, that you need the team on a Friday evening (which is weekend in Israel)?

Let’s follow up internally as if we didn’t miss something, something like this needs advance coordination and confirmation.

**Review by j9t — APPROVED** [review]

**Comment by j9t** [comment]
I’m approving this now, but only to help expedite. Fix the space issue before merging. Also contact me in Slack, please, on when this was communicated, and if not, why not. Thanks.

**Pull request merged**